Rezabekf/react md migration (#163)
aws-samples/cfn101-workshop: content/advanced/resource-types/example-in_Python/index.md

:::code{language=shell showLineNumbers=false showCopyAction=false}
cd aws-cloudformation-samples/
cd resource-types/awssamples-ec2-importkeypair/python/
:::

Let's take a look at a number of elements in the directory structure:

* `docs/`: contains auto-generated syntax information for properties of the resource type. Every time you make changes to the resource schema file, you want to refresh auto-generated code - that includes files in this directory - with the `cfn generate` CloudFormation CLI [command](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-cli-generate.html);
* `inputs/`: contains files with key/value data for resource type input properties. The resource type creator specifies this input information for use with contract tests: *do not add sensitive information to those files*. For more information, see [Specifying input data for use in contract tests](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-test.html#resource-type-test-input-data);
* `awssamples-ec2-importkeypair.json`: resource schema file, named after the chosen resource type name, used to **describe the model for the resource**;

> * `src/`: contains a directory named after the resource type name, inside which you should find:

Reviewer: ```suggestion
* `src/`: contains a directory named after the resource type name, inside you will find:
```

Author: done
